### ISSUE TITLE
All newlines were stripped in run only mode

### ISSUE THREAD

**Issue by tonidy** [nalgeon/codapi-js#29]
I've this problem when running python code in Docusaurus 3.3.2:

```python
def greet(name):
  print(f"Hello, {name}!")

greet("World")
```

```html
<codapi-snippet sandbox="python" init-delay="500">
</codapi-snippet>
```

And get this error:
<img width="912" alt="image" src="https://github.com/nalgeon/codapi-js/assets/823034/628e9be9-a1bc-491a-8967-9ff42d6aceef">

I've also seen this issue https://github.com/nalgeon/codapi-js/issues/22, @srenatus mentioned that all newlines are stripped in running only mode (without edit mode)
> <img width="745" alt="image" src="https://github.com/nalgeon/codapi-js/assets/823034/202dd5dd-b544-44c3-bcfd-2f7b41639056">

It should be great if this issue can be fixed. Thank you

**Comment by srenatus** [comment]
Can you put together an example on codesandbox, or share the concrete HTML you get for that snippet? I think we'll need that to assess what's happening here.

**Comment by tonidy** [comment]
@srenatus sure, here is the codesandbox URL https://codesandbox.io/p/devbox/peaceful-firefly-3sf4zz?workspaceId=273fe185-9274-4f80-b5dd-33b5233a4574. Then go to `docs/intro` or `intro.md` file.

I just pasted the snippet code to reproduce the issue:

````markdown
```python
def greet(name):
  print(f"Hello, {name}!")

greet("World")
```

<!-- Not working -->
<codapi-snippet sandbox="python" init-delay="500">
</codapi-snippet>

<!-- Working well -->
<!-- <codapi-snippet sandbox="python" editor="basic" init-delay="500">
</codapi-snippet> -->
````

**Comment by nalgeon** [comment]
#22 was fixed. What is your Codapi widget version?

**Comment by tonidy** [comment]
@nalgeon I'm using codapi widget 0.19.6. https://github.com/nalgeon/codapi-js/issues/22 only fixed when widget using edit mode, if widget disabled edit mode then it's not working

**Comment by srenatus** [comment]
FWIW this is the HTML I had been wondering about -- taken a screenshot from your code sandbox:
<img width="983" alt="image" src="https://github.com/nalgeon/codapi-js/assets/870638/ab29847b-1682-4c8f-8c5b-42ed5f50be71">

**Comment by nalgeon** [comment]
Yes, I understand, thank you. Will take a look at it when I have time.

**Comment by tonidy** [comment]
Great, thanks @nalgeon 

@srenatus do you mean `whitespace` inside HTML ?

**Comment by tonidy** [comment]
No worries, @srenatus. Even if it feels like you're muddying the water, your suggestion about checking the HTML output from Docusaurus might be valuable to @nalgeon 💪 .

**Comment by nalgeon** [comment]
Fixed in 0.19.7.

**Comment by tonidy** [comment]
@nalgeon awesome, it's working now. Thanks for fixing it!
